src/write/elf/object.rs

```diff
@@ -166,280 +170,175 @@ impl<'a> Object<'a> {
             return Ok(());
         };
 
+        let unsupported_reloc = || Err(Error(format!("unimplemented relocation {:?}", reloc)));
+
+        #[rustfmt::skip]
```

Comment:
```suggestion
```
I don't think manual formatting is necessary here, and it makes it more difficult to change this code in future. I think avoiding line wrapping is enough.